packages/cli/commands/init.js

```diff
@@ -125,17 +130,22 @@ exports.handler = async options => {
     logger.info(i18n(`${i18nKey}.logs.updateConfig`));
     process.exit(EXIT_CODES.ERROR);
   }
+  if (bothConfigFilesExist(useRootConfig)) {
+    logger.error(i18n(`${i18nKey}.errors.bothConfigFilesNotAllowed`));
+    process.exit(EXIT_CODES.ERROR);
+  }
 
   trackAuthAction('init', authType, TRACKING_STATUS.STARTED);
-  createEmptyConfigFile({ path: configPath });
+  createEmptyConfigFile({ path: configPath }, useRootConfig);
+  loadConfig(configPath, options);
   handleExit(deleteEmptyConfigFile);
 
   try {
     const { accountId, name } = await CONFIG_CREATION_FLOWS[authType](
       env,
       optionalAccount
     );
-    const configPath = getConfigPath();
+    const configPath = getRootOrDeprecatedConfigPath();
```

Comment:
Do we need to get config path a second time? Might make sense to rename if it has a different value

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1123/commits/7f9053a5b2f5659aad7c4203bd05278e774ed862